contracts/AirbroCampaignFactory.sol

```diff
@@ -13,6 +13,7 @@ import "./shared/AirdropBeta.sol";
 contract AirbroCampaignFactory is AirdropBeta {
     IAirdropRegistry public immutable airdropRegistryAddress;
     address public immutable treasury;
+    address public trustedRelayer;
```

Comment:
Not really necessary anymore

Reply:
@bxmmm1  wanted to ask, is it a security risk to not have a specific whitelisted address that is allowed to execute a function in the name of someone else?
In our case I'm guessing it's not a risk due to the merkle root/proof check when someone attempts to claim?

Reply:
In a situation like this, it doesn't really matter.
Nothing really changes based on msg.sender.
Airbro will get x amount of ETH, the intended recipient will get NFT as a reward, the only difference is that msg.sender will pay value + gas. instead of intended recipient.